amstrax/auto_processing/copy_live.py

```diff
@@ -0,0 +1,83 @@
+import subprocess
+import pymongo
+import amstrax
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Script that automatically copies new runs to stoomboot')
+    parser.add_argument(
+        '--detector',
+        type=str,
+        help='The detector that you are using',
+        default='xams')
+    parser.add_argument(
+        '--max_runs',
+        type=int,
+        help='How many runs you want to copy every time',
+        default=1)
+    args = parser.parse_args()
+    detector = args.detector
+    max_runs = args.max_runs
+
+    runsdb = amstrax.get_mongo_collection()
```

Comment:
*important*  please make sure you use the right collection for the detector argument, they (hopefully) use different collections!